### ISSUE TITLE
IntelliJ plugin does not accept legal .bazelignore files

### ISSUE THREAD

**Issue by dhalperi** [bazelbuild/intellij#1838]
Starting in version 2.1.0, `bazel` has allowed comments in the `.bazelignore` file. [commit](https://github.com/bazelbuild/bazel/commit/30f02c933f5f72e60f1dff98d5eab210eb41fb16).

However, when such a file is used with the IntelliJ plugin the project fails to sync. If your `.bazelignore` has a comment in it:

```
# a comment
some-package
```

The IntelliJ plugin will run something like this:

```
Querying targets in project directories...
Command: /usr/local/bin/bazelisk query [snip] - //# a comment/...:all - //some-package/...:all
```

Thus having a perfectly legal `.bazelignore` file will cause the project to be unusable in IntelliJ.

**Comment by dhalperi** [comment]
I already sent a (tested) PR fixing this issue in #1837. It's a one-liner exactly matching the original bazel one-line change. Obviously, I did not wait for discussion with maintainers, but it did not seem like much of a controversial change. Hopefully this is something we can discuss and decide to accept (or not) quickly.

**Comment by dhalperi** [comment]
Hey @brendandouglas - I was not going to ping for a few weeks, but I saw that you're active again on one of the issues I updated yesterday. I hope you had a good memorial day (if you're in US that is), as much as any of us had a "real" holiday in these times :).

I was wondering if there's anything I should do more to address this (and PR #1837). It does not seem like it ought to be controversial, and the functionality looks isolated from impacting Blaze.
